src/components/map/CuriosityMap.tsx

```diff
@@ -1,9 +1,26 @@
-import React from "react";
+import React, {useState} from "react";
+import './Map.scss'
 
 export const CuriosityMap: React.FunctionComponent = () => {
+  const [date, setDate] = useState('0')
+
+  function handleClick(day:string) {
+    setDate(day)
+    console.log(date)
+  }
   return (
-    <div>
-        CuriosityMap
-    </div>
+    <main>
+
```

Comment:
Ditto below, no stray blank lines (blank lines can be used to split up blocks of related code, but I don't think that applies here)
```suggestion
```